Dockerfile

```diff
@@ -300,7 +300,8 @@ COPY --from=python-plugins /usr/local/lib/python2.7/ /lib/python2.7
 COPY --from=python-plugins /usr/bin/python /bin/python
 
 COPY scripts/umount-hostfs-non-persistent /bin/umount-hostfs-non-persistent
-COPY deployments/docker/agent.yaml /etc/signalfx/agent.yaml
+ARG build_target=docker
+COPY deployments/${build_target}/agent.yaml /etc/signalfx/agent.yaml
```

Comment:
Did you mean to commit this?  I would rather it not be because people are going to have to override the fargate config in 99% of the cases to get useful application metrics so there is little point to making a build with some standard config.  It would be better to enhance the standard Docker agent.yaml file to work on both with envvar references used generously.